sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClientBuilder.java

```diff
@@ -305,6 +333,7 @@ public TableClientBuilder sasToken(String sasToken) {
      *
      * @throws NullPointerException If {@code credential} is {@code null}.
      */
+    @Override
```

Comment:
Setting a custom `HttpPipeline` would make it so the builder ignores the value set in this method. We should probably add a note like you did in other methods on this same class.